src/ScottPlot5/ScottPlot5/Fonts.cs

```diff
@@ -5,6 +5,37 @@
 /// </summary>
 public static class Fonts
 {
+    /* Typefaces are cached to improve performance.
+     * https://github.com/ScottPlot/ScottPlot/issues/2833
+     * https://github.com/ScottPlot/ScottPlot/pull/2848
+     */
+    private static readonly Dictionary<int, SKTypeface> TypefaceCache = new();
+
+    /// <summary>
+    /// Gets or sets the global font resolver
+    /// </summary>
+    private static IFontResolver? _fontResolver;
+    public static IFontResolver? FontResolver
+    {
+        get => _fontResolver;
+        set
```

Comment:
If the user changes the font resolver that should probably invalidate the typeface cache as well.

Reply:
To be honest, I don’t know what to say here. I thought about it. On the one hand, you are right, but on the other... I even had an idea to prohibit repeated changes the font resolver.

Reply:
I think a setter should just be a setter, if there's a good reason to add such behaviour then I think it would ideally be in a `UpdateFontResolver` method such that we can properly document its behaviour.

I also wonder what purpose this behaviour would have (beyond just a normal setter). I can't see a reason why it should be restricted in such a way, and if the user wants this kind of "set-if-not-exists" behaviour then they can do it themselves.

Reply:
I'll probably side with the simplest option (for now at least) and make this a basic getter/setter without a backing field. It's already nullable, so that signals to the user that it may be null (and also that newing classes may require disposal considerations on behalf of the one doing the newing)